AnnoyancesFilter/sections/css_extended.txt

```diff
@@ -97,6 +97,7 @@ facebook.com,facebookcorewwwi.onion#%#(function() { var observer=new MutationObs
 ! facebook.com#?#div[id^="substream_"] div[id^="hyperfeed_story_id_"][-ext-has='[id*="feed_subtitle_"] a[href^="https://www.facebook.com/#"] > span > span']
 ! End Facebook
 freundin.de#?#.region-infinite-block > .region-full-content:has( > .newsletter)
+kk.no#?#div[class]:has(> h1:contains(Vi bryr oss om ditt personvern))
```

Comment:
Move to cookies.

Reply:
Why not `kk.no##div[class^="PrivacyPopup"]`?

Reply:
I was trying the same rule first, but I checked another pages and found another code for cookie.
So, it works only on main page and that's why I chose ext-css.